app/src/lib/pocket.ts

```diff
@@ -79,7 +79,19 @@ export async function getAppsInNetwork(): Promise<Application[]> {
       break
     }
     response?.applications.forEach((app) => {
-      applicationsList.push(app)
+      const { address, chains, public_key: publicKey, jailed, max_relays: maxRelays, status,
+        staked_tokens: stakedTokens, unstaking_time: unstakingCompletionTime } = app.toJSON()
+
+      applicationsList.push({
+        address,
+        chains,
+        publicKey,
+        jailed,
+        maxRelays: BigInt(maxRelays),
+        status,
+        stakedTokens: BigInt(stakedTokens),
```

Comment:
Ah shoot, on this side we do need to keep things as a `BigInt`. We're not converting this to a number anywhere else right?